### PR TITLE
Add short url support

### PR DESCRIPTION
**Pull request by stanislavkocan**
### Short description

This PR adds link for each page in Pages (next to buttons View Page, Archive Page, ...)  which generates unique string of 5 characters and copies URL in format **url/s/<generated_string>** into clipboard. After user navigates to copied URL, it will redirect to corresponding page.

### Proposed changes

- There is link in Pages (next to View, Edit, Archive, Delete page) for each page with title Copy short link.
- In case page has no translation, link is visibly disabled with corresponding title.
- After click on that link for first time for some page, it generates random short string (using [shortuuid](https://pypi.org/project/shortuuid/) library), which is stored as new attribute called short_url_id  in AbstractBasePageTranslation class.
- Default value of attribute is empty string. New short_url_id is generated only when attribute has default value.
- Click on button also copies url in format **url/s/<generated_string>** into clipboard, with message informing user about successfuly copied url (using [pyperclip](https://pypi.org/project/pyperclip/) plugin).
- When user comes to copied short url, there is queryset which filters PageTranslation with given ID.
- User is redirected to page which is given by get_absolute_url() method

### Resolved issues

Fixes: #384

### CODE REVIEW

File: src/cms/views/pages/page_actions.py [hunk 1]
```diff
@@ -109,6 +113,51 @@ def delete_page(request, page_id, region_slug, language_code):
...
+        + "://"
+        + str(request.get_host())
+        + "/"
+        + str(page_translation.get_short_url())
+    )
+    pyperclip.copy(short_url)
+    messages.success(
+        request,
+        _("URL '{}' was successfuly copied to clipboard.".format(short_url)),
+    )
```
Comment (timoludwig) [inline src/cms/views/pages/page_actions.py:136]:
```suggestion
    try:
        pyperclip.copy(short_url)
        messages.success(
            request,
            _("URL '{}' was successfuly copied to clipboard.".format(short_url)),
        )
    # pylint: disable=broad-except
    except Exception as e:
        logger.exception(e)
        messages.warning(
            request,
            _("URL '{}' could not be copied to clipboard.".format(short_url)),
        )
```
I couldn't find any resources on the browser support of pyperclip. Even though it worked for me, I think we should at least print the short url even if the copy failed.

**Review by timoludwig — COMMENTED** [review]
Amazing, thank you for your contribution! :heart_eyes: :rocket: 

In general, your design decisions are very reasonable. See my suggestions below how you could improve your solution, but most of my comments are just my personal preferences regarding naming conventions.

**Comment by stanislavkocan** [comment]
Thank you @timoludwig for review. I fixed it in my last commit :)

I can see there are some conflicts that must be resolved, do I need to do it manully or you can merge it anyway ?

**Review by timoludwig — APPROVED** [review]
Thank you for your improvements and for adapting our style conventions! :+1:
Now, this PR is ready to merge :tada: 

I will change the base branch to an intermediate branch where I can resolve the conflicts and add the German translations of the hardcoded strings. I will let you know when the final changes are merged into the develop branch.

**Comment by timoludwig** [comment]
@stanislavkocan Your changes are now merged into develop :tada: 
Thanks again for your contribution :heart:

**Pull request merged**